host/sgx/sgxquoteprovider.h

```diff
@@ -25,7 +25,9 @@ typedef struct _oe_sgx_quote_provider
         free_sgx_quote_verification_collateral;
 } oe_sgx_quote_provider_t;
 
-bool oe_sgx_set_quote_provider_logger(sgx_ql_logging_function_t logger);
+// Set customized logging function for SGX provider.
+// This is for OE SDK internal tools only and could be removed in future.
```

Comment:
Isn't everything in this file for internal tools only?  If so, recommend moving line 29 up to line 3.

Reply:
Not very sure, in this file there are oe_ prefixed APIs, it looks like they are OE public APIs, maybe need a refactor to move out those APIs and make this private

Reply:
Yeah, public APIs and internal APIs should not be in the same file.   If there's a mix, at least file an issue to track splitting the file.

Reply:
That's fair, I created issue https://github.com/openenclave/openenclave/issues/4370 and set it to a bigger scope, we need some code re-organization. But I really like the idea of having clear boundaries between internal and public APIs